koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -147,14 +148,14 @@ def _generate_create_table_sql(self, partition_map=None):
                 f") WITH(external_location = 's3a://{s3_path}', format = 'PARQUET',"
                 " partitioned_by=ARRAY['source', 'year', 'month'])"
             )
-        LOG.info(f"Create Parquet Table SQL: {sql}")
+        LOG.info(log_json("Create Parquet Table SQL", table=self._table_name, schema=self._schema_name, sql=sql))
```

Comment:
```suggestion
        LOG.info(log_json(msg="create parquet table", table=self._table_name, schema=self._schema_name))
```
`msg` is a required kwarg. Let's also avoid logging the actual sql itself. I don't think anyone finds it beneficial.